### PR TITLE
chore: remove unnecessary checks for location hashes

### PR DESCRIPTION
**Pull request by iisakkirotko**
Hash is never included in `window.location.pathname`

### CODE REVIEW

**Comment by iisakkirotko** [comment]
* **#815** <a href="https://app.graphite.dev/github/pr/widgetti/solara/815?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#814** <a href="https://app.graphite.dev/github/pr/widgetti/solara/814?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#813** <a href="https://app.graphite.dev/github/pr/widgetti/solara/813?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @iisakkirotko and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by iisakkirotko** [comment]
* **#815** <a href="https://app.graphite.dev/github/pr/widgetti/solara/815?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#814** <a href="https://app.graphite.dev/github/pr/widgetti/solara/814?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#813** <a href="https://app.graphite.dev/github/pr/widgetti/solara/813?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @iisakkirotko and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by iisakkirotko** [comment]
* **#815** <a href="https://app.graphite.dev/github/pr/widgetti/solara/815?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#814** <a href="https://app.graphite.dev/github/pr/widgetti/solara/814?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#813** <a href="https://app.graphite.dev/github/pr/widgetti/solara/813?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @iisakkirotko and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: solara/widgets/vue/navigator.vue [hunk 1]
```diff
@@ -21,10 +21,6 @@ modules.export = {
       this.location = href;
     };
     let location = window.location.pathname.slice(solara.rootPath.length);
-    // take of the anchor
-    if (location.indexOf("#") !== -1) {
-      location = location.slice(0, location.indexOf("#"));
-    }
```
Comment (maartenbreddels) [inline solara/widgets/vue/navigator.vue:27]: But if you do a hard page navigation with an anchor in it, this will include a hash right?

Comment (maartenbreddels) [reply solara/widgets/vue/navigator.vue:27]: Ah, no, when I look at the anatomy: https://developer.mozilla.org/en-US/docs/Web/API/Location I see it does not include it

**Review by maartenbreddels — APPROVED** [review]

**Comment by iisakkirotko** [comment]
### Merge activity

* **Oct 24, 11:04 AM EDT**: A user started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/widgetti/solara/813).

**Pull request merged**